apps/studio/components/interfaces/Auth/AuthProvidersFormValidation.tsx

```diff
@@ -399,9 +399,8 @@ const EXTERNAL_PROVIDER_APPLE = {
       type: 'boolean',
     },
     EXTERNAL_APPLE_CLIENT_ID: {
-      title: 'Service ID (for OAuth)',
-      description: `Client identifier used in the OAuth flow on the web.
-[Learn more](https://developer.apple.com/documentation/sign_in_with_apple/configuring_your_environment_for_sign_in_with_apple)`,
+      title: 'Client IDs (Web, OAuth, iOS, macOS, watchOS, tvOS bundle IDs or service IDs)',
```

Comment:
<img width="591" alt="Screenshot 2024-11-04 at 17 53 24" src="https://github.com/user-attachments/assets/563642e3-430e-4489-ad79-04c90115a0ab">

this is an enormous label

Reply:
the content in parenthesis is probably a description, or optional label

Reply:
@MildTomato We can change it to: `Client IDs (Web, OAuth, Apple Platforms, or Service IDs)`

Reply:
agree with Jonny here - this label is really long and looks odd
+1 on moving it to description instead

Something like this feels better:
![image](https://github.com/user-attachments/assets/7c32e5f1-3816-4e7a-8617-9d7fe0654985)

Reply:
that's better